src/Autocomplete/index.tsx

```diff
@@ -100,14 +102,25 @@ const Autocomplete: React.FC<Props> = ({
   useEffect(() => {
     let suggestions: any[] = []
     if (!result.loading && result.data) {
-      const edges = _.get(result.data, `${connectionName}.edges`)
-      suggestions = _.map(edges, (edge: any) => ({
-        label: extractLabel(edge.node),
-        value: extractValue(edge.node),
-      }))
+      if (connectionName) {
+        const edges = _.get(result.data, `${connectionName}.edges`)
+        suggestions = _.map(edges, (edge: any) => ({
+          label: extractLabel(edge.node),
+          value: extractValue(edge.node),
+        }))
+      }
```

Comment:
Did you run prettier? This looks inconsistent.

Reply:
Doesn't prettier run on commit?